src/Component/src/Metadata/MetadataInterface.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Metadata;
+
+/**
+ * @method string|null getStateMachineComponent()
```

Comment:
Method not added to interface due BC? Is it safe to add now and add notice to UPGRADE? Since method exists on `Metadata`, covered with spec and probably used.

Reply:
yeah, cause I think it will be removed in 2.x of this package, I don't think it's useful to do it now.